src/pages/orders/orders-page/orders-page.js

```diff
@@ -17,6 +18,7 @@ import { config } from '../../../configs';
 
 const OrdersPage = () => {
   const styles = useStyles();
+  const common = useCommonStyles();
```

Comment:
Can you rename this `common` variable to styles or classes?

Reply:
fixed